docs/source/configuration.rst

```diff
@@ -55,6 +55,13 @@ line switch, as seen here:
                                      (default), no Git object paths are excluded
                                      unless effectively excluded via the
                                      --include-paths option.
+     -a, --allow-signatures TEXT     Specify signatures of matches that you
+                                     explicitly want to allow, and mark as okay.
+                                     These signatures are generated during the
+                                     scan process, and reported out with each
+                                     individual match. This option can be
+                                     specified multiple times, to allow as many
+                                     signatures as you would like.
```

Comment:
Maybe a nitpick, but `allow-signatures` is somewhat ambiguous to me. `exclude-signatures` would be more explicit, IMO. Otherwise, beyond what was mentioned, looks good to me.

Reply:
I'd have no problem renaming this if there's a consensus. `allow-signatures` is what we settled on in discussions on #2. Any thoughts here @rscottbailey or @decompil3d ?

Reply:
Sure, `exclude-signatures` seems fine to me.

Reply:
I'm fine with `exclude-signatures` as well.